app/Svg/Parser.hs

```diff
@@ -38,78 +38,80 @@ import Data.List.Split (splitOn)
 parsePrebuiltSvgs :: IO ()
 parsePrebuiltSvgs = runSqlite databasePath $ do
     deleteGraphs
-    performParse "Computer Science" "csc2020.svg"
-    performParse "(unofficial) Statistics" "sta2017.svg"
-    performParse "(unofficial) Biochemistry" "bch2015.svg"
-    performParse "(unofficial) Cell & Systems Biology" "csb2015.svg"
-    performParse "(unofficial) Estonian" "est2015.svg"
-    performParse "(unofficial) Finnish" "fin2015.svg"
-    performParse "(unofficial) Italian" "ita2015.svg"
-    performParse "(unofficial) Linguistics" "lin2015.svg"
-    performParse "(unofficial) Rotman" "rotman2015.svg"
-    performParse "(unofficial) Economics" "eco2015.svg"
-    performParse "(unofficial) Spanish" "spa2015.svg"
-    performParse "(unofficial) Portuguese" "prt2015.svg"
-    performParse "(unofficial) Slavic" "sla2015.svg"
-    performParse "(unofficial) East Asian Studies" "eas2015.svg"
-    performParse "(unofficial) English" "eng2015.svg"
-    performParse "(unofficial) History and Philosophy of Science" "hps2015.svg"
-    performParse "(unofficial) History" "his2015.svg"
-    performParse "(unofficial) Geography" "ggr2015.svg"
-    performParse "(unofficial) Aboriginal" "abs2015.svg"
-    performParse "(unofficial) German" "ger2015.svg"
+    mapM_ (uncurry performParseFromFile)
```

Comment:
just some style changes

Reply:
Don't mix style or refactoring changes and functionality changes in the same PR. As far as I can tell, the changes in this file from here to 113 are only style changes, and so should be pulled out into a separate PR.

Also I don't like inserting whitespace to vertically align list arguments (on the line below). Use a single space after each comma.

Reply:
ok i reverted it back to the original